tools/cli.js

```diff
@@ -24,9 +24,17 @@ const log = (msg) => {
   if (userInput) rl.write(userInput);
 };
 
-function complete(input, completions) {
-  if (!input) return completions;
-  return completions.filter(c => c.startsWith(input));
+function complete(input, completions, forbidden = []) {
+  if (!forbidden && !input) return completions;
+  return completions.filter(
+    c => !startsWithAny(c, forbidden) && (!input || c.startsWith(input)));
+}
+
+function startsWithAny(str, array) {
+  for (const el of array) {
```

Comment:
I'd just do `array.some(el => str.startsWith(el))`, but this imperative approach is fine too ;)